src/screens/CreatePost/index.js

```diff
@@ -249,39 +247,13 @@
     }
   ];
 
-  const fetchMyProfile = async () => {
-    setLoading(true);
-    const userId = await getUserId();
-    if (userId) {
-      const result = await getMyProfile(userId);
-      if (result.code === 200) {
-        setDataProfile(result.data);
-        setLoading(false);
-        handleLocation(result.data);
-      }
-
-      setLoading(false);
-    }
-  };
-
-  React.useEffect(() => {
-    getSpecificCache(PROFILE_CACHE, async (res) => {
-      if (!res) {
-        fetchMyProfile();
-      } else {
-        setDataProfile(res);
-        handleLocation(res);
-      }
-    });
-  }, []);
-
   const handleLocation = async (res) => {
     setGeoList([...location, ...res?.locations]);
     setLoading(false);
   };
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `handleLocation` function has been modified to no longer require a parameter and is now being called inside a `useEffect` hook. Ensure that the `res` object is being correctly handled within the function itself.
```diff
+ const handleLocation = async (res) => {
+   setGeoList([...location, ...res?.locations]);
+   setLoading(false);
+ };
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->